processor/src/chiplets/mod.rs

```diff
@@ -219,6 +244,16 @@ impl Chiplets {
         self.bus.send_queued_hasher_request(self.clk);
     }
 
+    // HASH CHIPLET HELPERS
+    // --------------------------------------------------------------------------------------------
+
+    /// Provides hasher lookups to the Chiplets Bus
+    fn provide_hasher_lookups(&mut self, lookups: &[HasherLookup]) {
+        for lookup in lookups.iter() {
+            self.bus.provide_hasher_lookup(*lookup, lookup.cycle());
+        }
+    }
+
```

Comment:
I wonder if it makes more sense to have a bulk provide function in the bus instead? Then we could just call `self.bus.provide_hasher_lookups` and keep all of the request/provide methods in one place. Is there a reason you decided not to take this approach?

Reply:
No reason, I agree that keeping it at one place seems better. Moved it to the bus. Thanks.